src/mongoConnection.py

```diff
@@ -50,5 +53,9 @@ def count_data_from_mongo(query=None):
     except Exception as e:
         print(f'Fehler beim lesen der Daten: {e}')
         return None
-    finally:
```

Comment:
Was hattest du hier vor?

Reply:
@felixkru es gibt eine eigenen Client Close-Funktion, wenn ich eine query mehrmals ausführe z.B. in einer Schleife ist das nicht möglich wenn der Client bei jeder Anfrage geschlossen wird.